rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -145,10 +149,36 @@ def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False,
         # Image sizes of a batch
         self.img_size = np.zeros((self.bs * 2), dtype="int32")
 
+    def get_cupy_array_from_address(self):
+        if(self.tensor_format == types.NCHW):
+            address = self.loader.copyToHipTensorNCHW(
+                self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
+            print("Address : ",address)
```

Comment:
comment out this print command if it is needed only for debug

Reply:
Done